website/docs/guides/getting-started.html.markdown

```diff
@@ -0,0 +1,592 @@
+
+The Replication Controller allows you to replicate pods. This is useful
+for maintaining overall availability and scalability of your application
+exposed to the user.
+
+We can just replace our Pod with RC from the previous config
+and keep the Service there.
+
+```hcl
+resource "kubernetes_replication_controller" "nginx" {
+  metadata {
+    name = "scalable-nginx-example"
+    labels {
+      App = "ScalableNginxExample"
+    }
+  }
+
+  spec {
+    replicas = 2
+    selector {
+      App = "ScalableNginxExample"
+    }
+    template {
+      container {
+        image = "nginx:1.7.8"
+        name  = "example"
+
+        port {
+          container_port = 80
+        }
+
+        resources {
+          limits {
+            cpu    = "0.5"
+            memory = "512Mi"
+          }
+          requests {
+            cpu    = "250m"
+            memory = "50Mi"
+          }
+        }
+      }
+    }
+  }
+}
+
+resource "kubernetes_service" "nginx" {
+  metadata {
+    name = "nginx-example"
+  }
+  spec {
+    selector {
+      App = "${kubernetes_replication_controller.nginx.metadata.0.labels.App}"
+    }
+    port {
+      port = 80
+      target_port = 80
+    }
+
+    type = "LoadBalancer"
+  }
+}
+```
+
+You may notice we also specified how much CPU and memory do we expect
+single instance of that application to consume. This is incredibly
+helpful for K8S as it helps avoiding under-provisioning or over-provisioning
+that would result in either unused resources (costing money) or lack
+of resources (causing the app to crash or slow down).
+
+```
+$ terraform plan
+
+...
+
+Plan: 2 to add, 0 to change, 0 to destroy.
+```
+
+```
+$ terraform apply
+
+kubernetes_replication_controller.nginx: Creating...
+...
+kubernetes_replication_controller.nginx: Creation complete (ID: default/scalable-nginx-example)
+kubernetes_service.nginx: Creating...
+...
+kubernetes_service.nginx: Still creating... (10s elapsed)
+kubernetes_service.nginx: Still creating... (20s elapsed)
+kubernetes_service.nginx: Still creating... (30s elapsed)
+kubernetes_service.nginx: Still creating... (40s elapsed)
+kubernetes_service.nginx: Still creating... (50s elapsed)
+kubernetes_service.nginx: Creation complete (ID: default/nginx-example)
+
+Apply complete! Resources: 2 added, 0 changed, 0 destroyed.
+
+The state of your infrastructure has been saved to the path
+below. This state is required to modify and destroy your
+infrastructure, so keep it safe. To inspect the complete state
+use the `terraform show` command.
+
+State path:
+
+Outputs:
+
+lb_ip = 35.197.9.247
+```
+
+Unlike in previous example, the IP address here will direct traffic
+to one of the 2 pods scheduled in the cluster.
+
+### Updating Configuration
+
+As our application user-base grows we might need more instances to be scheduled.
+The easiest way to achieve this is to increase `replicas` field in the config
+accordingly.
+
+```hcl
+resource "kubernetes_replication_controller" "example" {
+...
```

Comment:
Totally minor, but I suggest using an indented `# ...` for "etc" placeholders, since it co-operates better with the syntax highlighting.